python/lsst/analysis/tools/interfaces/_task.py

```diff
@@ -351,6 +370,10 @@ def runQuantum(
             `lsst.daf.butler.DatasetRef` objects associated with the defined
             output connections.
         """
+        # TODO: This rcParams modification is a temporary solution, hiding
+        # a matplotlib warning indicating too many figures have been opened.
+        # When DM-39114 is implemented, this should be removed.
+        plt.rcParams.update({"figure.max_open_warning": 0})
```

Comment:
It seemed like the best place for such a catch-all config at the time, however, I'm not beholden to it. Is it causing issues elsewhere?

Reply:
No issues yet, but this method is overridden in a few tasks and they miss this. I think because they are an exception instead of the norm, it doesn't lead to too many open figures but I figured it might be best places elsewhere. I'm working on a ticket that generalizes this method further and can move it on that ticket.